votes/models.py

```diff
@@ -1879,3 +1884,47 @@ def get_same_day_decision_navigation(decision: DecisionType) -> NavigationResult
             previous_decision = same_day_decisions[current_index + 1]
 
     return NavigationResult(next_decision, previous_decision)
+
+
+class BulkAPIUser(DjangoVoteModel):
+    """
+    Model to track users who have access to bulk API data via token authentication.
+    """
+
+    email: EmailField
+    token: str = ""
+    purpose: TextField = ""
+    enabled: bool = True
+    access_count: PositiveIntegerField = 0
+    created_at: datetime.datetime
```

Comment:
fwiw, I think leaving this as an auto field with `auto_now_add` instead would be fine